hack/prepare.sh

```diff
@@ -0,0 +1,18 @@
+rm -rf cluster-deployment-automation
+
+CDA_REPO=${CDA_REPO:-https://github.com/bn222/cluster-deployment-automation.git}
+CDA_BRANCH=${CDA_BRANCH:-main}
+
+git submodule init
+git submodule update
+
+cd cluster-deployment-automation
+git checkout "$CDA_BRANCH"
+
+sed -e 's/#\(user\|group\) = ".*"$/\1 = "root"/' -i /etc/libvirt/qemu.conf
```

Comment:
This has been moved into cda. Can you also update the README.md in cda ?

Reply:
I removed it... now we can merge and we can open a pr to update readme in cda.